docs/source/learn/eager.md

```diff
@@ -100,33 +118,53 @@ def step_fn(model, data, target, loss_fn, optimizer):
 step_fn = torch_xla.compile(step_fn)
 ```
 
-In training we asked user to refactor the `step_fn` out because it is
-usually better to compile the model's forward, backward and optimizer
-together. The long term goal is to also use `torch.compile` for training
-but right now we recommend user to use `torch_xla.compile`(for
-perfomrance reason).
-
-## Benchmark
-
-I run a 2 layer decoder only model training(it is pretty much just a
-llama2) with fake data on a single chip of v4-8 for 300 steps. Below is
-the number I observed.
+- `torch.compile` is PyTorch's general-purpose compilation API designed to
+accelerate PyTorch models across various backends. For PyTorch/XLA, it uses the
+`openxla` backend. We recommend `torch.compile` for PyTorch/XLA inference
+because it lowers tracing overhead, leading to more efficient static inference
+graphs. To use it with XLA, simply specify `backend="openxla"`.
 
-  Mode                        token/s
-  --------------------------- ---------
-  Tracing mode (base line)    147
-  Eager mode                  65
-  Eager + torch_xla compile   147
-
-  : Eager mode benchmarks
+``` python
+torch_xla.experimental.eager_mode(True)
+compiled_model = torch.compile(model, backend="openxla")
+```
 
-Eager mode can achieve ~45% performance of the fully compiled model for
-the decoder only model. For more information, see
+The long-term aim is for `torch.compile` to be the single compilation API for
+both training and inference on XLA.
+
+## Performance Benchmarks
+
+To quantify the performance impact of torch_xla.compile and eager mode,
+benchmarks were conducted under specific conditions. The benchmarks utilized a
+2-layer decoder-only model, similar to Llama2, trained with fake data. The
+training process spanned 300 steps on a single chip of a v4-8 TPU. The observed
+performance, measured in tokens per second, clearly illustrates the impact of
+different execution modes:
+
+| Mode                        | token/s |
+|-----------------------------|---------|
+| Tracing mode (base line)    | 147     |
+| Eager mode                  | 65      |
+| Eager + torch_xla compile   | 147     |
+
+The benchmark results unequivocally demonstrate that eager mode combined with
+`torch_xla.compile` achieves performance parity with the traditional LazyTensor
+tracing mode, both yielding `147` tokens/s. This empirically validates the claim
```

Comment:
"... This shows that the new API provides a better user experience without a performance penalty.  " I'm trying to keep sentence length down to ease readibility. I don't think the last phrase ("no-regret ...") is needed.